src/nbl/asset/utils/ISPIRVOptimizer.cpp

```diff
@@ -92,3 +92,8 @@ nbl::core::smart_refctd_ptr<ICPUBuffer> ISPIRVOptimizer::optimize(const ICPUBuff
 
     return optimize(spirv, count, logger);
 }
+
+const std::initializer_list<ISPIRVOptimizer::E_OPTIMIZER_PASS> nbl::asset::ISPIRVOptimizer::getPasses() const
+{
```

Comment:
why not inline, and why `initializer_list` !?

Reply:
The optimizer holds an initializer list iirc. Apparently it's a light wrapper around a pointer + size a la `std::span`, so I just returned that (according to cppref the copy constructor does a shallow copy).

Reply:
eh, we need to fix and use a vector... cause initializer_list is only meant for arrays of compile time constants